### PR TITLE
Add clinical-delivery upload for raredisease

### PR DESCRIPTION
**Pull request by rannick**
### Added

- Add clinical-delivery upload for raredisease

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by rannick** [comment]
Tested successfully with amusingmarmoset

File: cg/meta/upload/raredisease/raredisease.py [hunk 1]
```diff
@@ -31,8 +31,12 @@ def upload(self, ctx: click.Context, case: Case, restart: bool) -> None:
         """Uploads RAREDISEASE analysis data and files."""
         analysis: Analysis = case.analyses[0]
         self.update_upload_started_at(analysis=analysis)
+
         ctx.invoke(upload_observations_to_loqusdb, case_id=case.internal_id)
 
+        # Clinical delivery upload
+        ctx.invoke(upload_clinical_delivery, case_id=case.internal_id)
```
Comment (ivadym) [inline cg/meta/upload/raredisease/raredisease.py:38]: should we add the delivery report generation to the uploads as well?

**Review by ivadym — APPROVED** [review]
💯

File: cg/constants/housekeeper_tags.py [hunk 2]
```diff
@@ -203,6 +203,18 @@ class BalsamicProtectedTags:
...
+        ["vcf-str"],
+        ["vcf-snv"],
+        ["qc-metrics"],
+        ["smn-calling"],
+        ["vcf-sv"],
+        ["vcf-sv-clinical"],
+        ["vcf-sv-research"],
+        ["vcf2cytosure"],
+        [HK_DELIVERY_REPORT_TAG],
+        [AnalysisTag.MULTIQC_HTML],
```
[the remark this answers is not in the document]
Comment (rannick) [reply cg/constants/housekeeper_tags.py:217]: Indeed!

File: cg/meta/upload/raredisease/raredisease.py (hunk 1 above)
Comment (rannick) [reply cg/meta/upload/raredisease/raredisease.py:38]: I can add it with the last command (genotype) and make the big test then?

Comment (ivadym) [reply cg/meta/upload/raredisease/raredisease.py:38]: sounds good 👍

**Pull request merged**